apex.js/apex.js-tests.ts

```diff
@@ -0,0 +1,5 @@
+import λ from "apex.js";
+
+const handler = λ((event: any) => {
```

Comment:
Because of TypeScript's [function bivariance](https://www.typescriptlang.org/docs/handbook/type-compatibility.html), avoid annotating the types of callbacks manually. (Especially with `any`, which turns off type-checking.)